### PR TITLE
Merge develop 8.2 30.08.17

### PR DESCRIPTION
**Pull request by livecodeali**
Conflict resolution: https://gist.github.com/livecodeali/2c4239e05b38934bb475e5d6d003bd3d

### CODE REVIEW

File: docs/dictionary/command/import-snapshot.lcdoc [hunk 1]
```diff
@@ -110,6 +110,65 @@ as the Tools palette).
 The format of the resulting <image> depends on the current setting of the
 <paintCompression> <property>.
 
+<<<<<<< HEAD
```
Comment (runrevmark) [inline docs/dictionary/command/import-snapshot.lcdoc]: There's a conflict marker here

**Comment by runrevmark** [comment]
@livecodeali Looks sound apart from the conflict marker in the commented-on doc entry.

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 651dec4

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok 651dec4fbc53267d59aa47bcf66de0760f0059fa

**Comment by livecode-vulcan** [comment]
😎 test success 651dec4fbc53267d59aa47bcf66de0760f0059fa

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/682)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/698)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/672)
* try-community-universal-ios-iphoneos10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos10.3/builds/615)
* try-community-universal-ios-iphonesimulator10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator10.3/builds/618)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/676)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/696)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/681)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/698)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/680)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/691)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/713)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/677)

**Pull request merged**